### ISSUE TITLE
`Duplicate key entity.minecraft.pig`

### ISSUE THREAD

**Issue by SettingDust** [AFunProject/Reputation#5]
What's the reason of this?
```
[15:37:05] [Render thread/INFO] (Reputation) Read in entity entity.minecraft.pig with 2 icon sets
[15:37:05] [Render thread/INFO] (Reputation) Read in entity entity.minecraft.pig with 1 icon sets
[15:37:05] [Render thread/ERROR] (Reputation) 'Duplicate key entity.minecraft.pig (attempted merging values mods.thecomputerizer.reputation.common.ai.ServerTrackers$Data@1a3d9aaf and mods.thecomputerizer.reputation.common.ai.ServerTrackers$Data@22116efc)'
java.lang.IllegalStateException: Duplicate key entity.minecraft.pig (attempted merging values mods.thecomputerizer.reputation.common.ai.ServerTrackers$Data@1a3d9aaf and mods.thecomputerizer.reputation.common.ai.ServerTrackers$Data@22116efc)
    INTERNALS
    at TRANSFORMER/reputation@0.9.9/mods.thecomputerizer.reputation.common.ai.ServerTrackers.parseChatIcons(ServerTrackers.java:36)
    at TRANSFORMER/reputation@0.9.9/mods.thecomputerizer.reputation.util.FactionListener.apply(FactionListener.java:70)
    at TRANSFORMER/reputation@0.9.9/mods.thecomputerizer.reputation.util.FactionListener.m_5787_(FactionListener.java:29)
    at TRANSFORMER/minecraft@1.18.2/net.minecraft.server.packs.resources.SimplePreparableReloadListener.m_10789_(SimplePreparableReloadListener.java:13)
```

**Comment by SettingDust** [comment]
Gotcha. The pig will be the default value of the type when lack of the entity

**Comment by TheComputerizer** [comment]
Reopening this until 1.0.0 actually gets released

**Comment by TheComputerizer** [comment]
Fixed in 1.0.0
